Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN add-apt-repository -y ppa:ondrej/php && \
         ${ps}-readline \
         ${ps}-xml \
         ${ps}-zip \
+        ${ps}-dev \
```

Comment:
This is needed for `phpize` on line 50.  We don't need this package to operate the website, we remove it, we also remove `git` and `python3`.